instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import './CommentSection.css';
+
+class CommentSection extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            likes : 0,
+            liked : false,
+            newComment : { username: localStorage.getItem('username'), text: ""},
+            updatedComments : this.props.comments
+        }
+    }
+
+    componentDidMount() {
+        this.setState(prevState => { return {
+            likes: this.props.likes, 
+            updatedComments : JSON.parse(localStorage.getItem(`comments${this.props.id}`)),
+            newComment : { username: localStorage.getItem('username'), text: ""}
+        }});
+    }
+
+    inputChange = e => {
+        this.setState({ newComment : {username: localStorage.getItem('username'), text: e.target.value}});
+    }
+
+    commentCreation = e => {
+        e.preventDefault();
+        if(this.state.newComment.text === "") { return 'No text to post' }
```

Comment:
I like that you added input validation. Consider this change to notify the user.
```suggestion
        if (this.state.newComment.text === "") {
          return alert("No text to post");
        }
```